pitop/pma/ultrasonic_sensor.py

```diff
@@ -65,56 +51,18 @@ def __init__(
                                     "max_distance": lambda: self.max_distance,
                                     "threshold_distance": lambda: self.threshold_distance})
 
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.close()
```

Comment:
we should add an example of using this to switch the use of a port

Reply:
Yeah I think gpiozero have some explanations on that too, only really useful if you're using python interactively though